src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -246,212 +253,142 @@ private String getInputArguments() {
      * 7Csort:date/jenkinsci-dev/AFYHSG3NUEI/UsVJIKoE4B8J
      * 
      */
-    public static class PdfArtifact extends Artifact {
-        private String pdfReportFilePath;
+    public static class PdfArtifact extends AbstractArtifactImpl {
 
-        @DataBoundConstructor
-        public PdfArtifact() {
-           
-        }
+        static final String PDF_REPORT_PATH = "PDFReportPath";

```

Comment:
why not make all these private static final?

Reply:
Done.